platform-configs/protectli-vp2420.robot

```diff
@@ -18,6 +19,7 @@ ${PAYLOAD_STRING}=                                  ${EMPTY}
 ${EDK2_IPXE_STRING}=                                Network Boot and Utilities
 ${EDK2_IPXE_CHECKPOINT}=                            Advanced
 ${EDK2_IPXE_START_POS}=                             2
+${IPXE_BOOT_ENTRY}=                                 Network Boot and Utilities
```

Comment:
I checked some of the variables and wrote down what values they take:
IPXE_BOOT_ENTRY = `iPXE Network boot` `Network Boot and Utilities` `${EMPTY}`
EDK2_IPXE_STRING = `iPXE Network Boot` `Network Boot and Utilities` `${EMPTY}`
IPXE_STRING = `Network Boot Firmware` `${EMPTY}`
IPXE_STRING2 = `${EMPTY}`
IPXE_KEY = `${EMPTY}`
NET_BOOT_KEY = `${EMPTY}`
EDK2_IPXE_CHECKPOINT = `iPXE Shell` `Advanced` `${EMPTY}`
EDK2_IPXE_START_POS = `1` `2` `${EMPTY}`

I replaced `IPXE_BOOT_ENTRY` with `EDK2_IPXE_STRING`
I remove the following variables because they are not used:  `IPXE_STRING2` `IPXE_KEY` `NET_BOOT_KEY`

Reply:
I would keep the IPXE_BOOT_ENTRY and remove EDK2_IPXE_STRING, because IPXE_BOOT_ENTRY is more descriptive (the name suggest what it is, while IPXE_STRING does not say anything)

Reply:
@miczyg1 
Done in ab4b08f5c066c4335022c2a2c1c8c0a70b43d1ba

Reply:
I confirm this change looks good.